http.go

```diff
@@ -144,6 +152,59 @@ func (h *xDSHandler) handleConfig(w http.ResponseWriter, req *http.Request) {
 	w.Write(j)
 }
 
+func (h *xDSHandler) handleBootstrap(w http.ResponseWriter, req *http.Request) {
+	if req.Method != "GET" {
+		http.Error(w, "method not allowed", 405)
+		return
+	}
+
+	clusterValues, ok := req.URL.Query()["cluster"]
+	if !ok || len(clusterValues) != 1 {
+		http.Error(w, "invalid cluster parameter", 400)
+		return
+	}
+
+	idValues, ok := req.URL.Query()["id"]
+	if !ok || len(idValues) != 1 {
+		http.Error(w, "invalid id parameter", 400)
+		return
+	}
+
+	node := &core.Node{
+		Id:      idValues[0],
+		Cluster: clusterValues[0],
+	}
+
+	configSnapshot := h.controller.configStore.GetConfigSnapshot()
+
+	clusterData, ok := configSnapshot.GetClusters(node)
+	if !ok {
+		http.Error(w, "not found", 404)
```

Comment:
Any of these `http.Error` calls need to be followed by a `return`, otherwise execution is going to continue.